traits/tests/test_trait_notifiers.py

```diff
@@ -0,0 +1,38 @@
+# (C) Copyright 2005-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
+from traits.api import HasTraits, Int
```

Comment:
Please follow the usual convention for import layout - there should be a blank line between import groups. `import unittest` belongs to the standard library import group, while the `from traits` import belongs to the 1st party imports.